tipg/dbmodel.py

```diff
@@ -489,9 +519,9 @@ def _where(
                             logic.Func(
                                 "ST_MakeEnvelope",
                                 tilebox.left,
-                                tilebox.bottom,
+                                max(-85, tilebox.bottom),
                                 tilebox.right,
-                                tilebox.top,
+                                min(85, tilebox.top),
```

Comment:
we could `truncate` the lat/lon directly with morecantile `truncate_lnglat` method (so it will work for all TMS)

Reply:
:eyes: I did not know such a method existed!

Reply:
👍 so I guess we can also use it at L335